source/features/sticky-file-header.css

```diff
@@ -5,7 +5,7 @@
 	top: 0;
 }
 
-.notification-shelf ~ .application-main .file-header {
+.notification-shelf ~ .application-main .file-header:not(.sticky-file-header) {
 	top: 129px;
 }
```

Comment:
Apparently GitHub uses JS to compute and inject the correct value for `top` in each of the file header elements' `style` attribute :grimacing: 

This rule was superseded by those inline styles but still messed with their computations, resulting in a value of `198px` instead of the correct `129px`.